lib/ui/painting.dart

```diff
@@ -1737,6 +1753,21 @@ class Image {
     return _image.toByteData(format: format);
   }
 
+  /// The color space that used by the [Image]'s colors. Note that on certain
```

Comment:
separate the first line (brief description) from the details

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements

Reply:
done